bin/git-forgit

```diff
@@ -947,55 +948,100 @@ export FORGIT_GI_REPO_REMOTE=${FORGIT_GI_REPO_REMOTE:-https://github.com/dvcs/gi
 export FORGIT_GI_REPO_LOCAL="${FORGIT_GI_REPO_LOCAL:-${XDG_CACHE_HOME:-$HOME/.cache}/forgit/gi/repos/dvcs/gitignore}"
 export FORGIT_GI_TEMPLATES=${FORGIT_GI_TEMPLATES:-$FORGIT_GI_REPO_LOCAL/templates}
 
-_forgit_ignore_preview() {
+_forgit_path_preview() {
```

Comment:
is it possible to split the refactor and the feature into two commits? That may make it easier to review and also help us chase down a regression easier

Reply:
Sure, done :slightly_smiling_face: